plugins/magento2/package.json

```diff
@@ -85,42 +33,25 @@
     "@types/jest": "^23.3.2",
     "@types/node": "^10.17.5",
     "@types/react": "^16.9.11",
-    "@types/shopify-buy": "^2.10.4",
     "colors": "^1.3.2",
     "commitizen": "^3.0.0",
     "coveralls": "^3.0.2",
     "cross-env": "^5.2.0",
     "cz-conventional-changelog": "^2.1.0",
     "esbuild": "^0.12.15",
     "husky": "^1.0.1",
-    "i": "^0.3.6",
-    "jest": "^29.5.0",
-    "jest-config": "^29.5.0",
-    "lint-staged": "^13.2.1",
     "lodash.camelcase": "^4.3.0",
     "prettier": "^1.14.3",
     "prompt": "^1.0.0",
     "replace-in-file": "^3.4.2",
     "rimraf": "^2.6.2",
     "rollup": "^2.53.2",
-    "rollup-plugin-dts": "^3.0.2",
     "rollup-plugin-esbuild": "^4.5.0",
-    "rollup-plugin-node-globals": "^1.4.0",
-    "rollup-plugin-node-polyfills": "^0.2.1",
     "rollup-plugin-replace": "^2.2.0",
     "rollup-plugin-serve": "^1.0.1",
-    "semantic-release": "^21.0.1",
-    "shelljs": "^0.8.3",
-    "travis-deploy-once": "^5.0.9",
-    "ts-jest": "^23.10.2",
-    "ts-node": "^7.0.1",
-    "tslint": "^5.11.0",
-    "tslint-config-prettier": "^1.15.0",
-    "tslint-config-standard": "^8.0.1",
-    "typedoc": "^0.12.0",
```

Comment:
amazing. removing all of these for all plugins will drastically reduce dependabot alerts eventually 🙏🏽